### DISCUSSION TITLE
Question about ID

### DISCUSSION THREAD

**Discussion by PabloDevGit** [Poing-Studios/godot-admob-editor#26] [General]
Hi, I'm not sure if this is the place to ask this question or if the plugin is involved in this issue.
Is this plugin updated to meet this google condition?

https://support.google.com/googleplay/android-developer/answer/6048248?hl=en

![Captura](https://user-images.githubusercontent.com/19652075/166166957-cb85acc5-c676-4a57-96ad-74abd8198e08.PNG)

**Comment by gumaciel** [comment]
Hello, as i read you can do it by yourself here:

![image](https://user-images.githubusercontent.com/20030153/166331007-0fffef26-cdde-4a96-9bfd-a80199f4a1d2.png)


Read more here: https://developers.google.com/admob/android/quick-start?hl=en#import_the_mobile_ads_sdk


Update this file on your project: ```res://android/build/AndroidManifest.xml```

**Reply by gumaciel** [reply]
Update: I think you don't need to do anything because we are already using the version 20.4.0:

https://github.com/Poing-Studios/godot-admob-android/blob/master/admob/build.gradle#L30
